### PR TITLE
Note on luhn description for single digits

### PR DESCRIPTION
**Pull request by stevejb71**
Add notes on edge cases for Luhn in the description, and add a 0 length string test.

### CODE REVIEW

File: exercises/luhn/description.md [hunk 1]
```diff
@@ -7,8 +7,11 @@ The task is to write a function that checks if a given string is valid.
 
 Validating a Number
 ------
+Strings of length 1 or less are not valid. Spaces are allowed in the input,
+but they should be stripped before checking. All other non-digit characters
+are disallowed.
 
-As an example, here is a valid (but fictitious) Canadian Social Insurance
+As an example of a valid string, here is fictitious Canadian Social Insurance
```
Comment (petertseng) [inline exercises/luhn/description.md]: I think "is a" rather than just "is"

File: exercises/luhn/description.md [hunk 2]
```diff
@@ -7,8 +7,11 @@ The task is to write a function that checks if a given string is valid.
 
 Validating a Number
 ------
+Strings of length 1 or less are not valid. Spaces are allowed in the input,
+but they should be stripped before checking. All other non-digit characters
```
Comment (petertseng) [inline exercises/luhn/description.md]: this sentence makes me think of other cases: how about a string of only spaces?

However, it's not true that I would refuse to see this merged unless such a case was added. What do people think of it?

File: exercises/luhn/canonical-data.json [hunk 3]
```diff
@@ -1,7 +1,12 @@
 {
   "valid": [
     {
-      "description": "single digit strings can not be valid",
+      "description": "zero length strings are not valid",
+      "input": "",
+      "expected": false
```
Comment (Insti) [inline exercises/luhn/canonical-data.json]: Is there an implementation that would return the wrong result for this case?

File: exercises/luhn/description.md (hunk 2 above)
Comment (stevejb71) [inline exercises/luhn/description.md]: I think the space stripping is already tested, so that a spaces only test would be superfluous.

File: exercises/luhn/canonical-data.json (hunk 3 above)
Comment (stevejb71) [inline exercises/luhn/canonical-data.json]: Probably not, the single character test cases should cover this. I'm happy to remove this commit unless anyone else can think of an example...

**Comment by stevejb71** [comment]
Removed the commit containing an extra test for 0 length string.
